src/components/Flashcard/CardContent.js

```diff
@@ -1,45 +1,77 @@
 import React, { useState, useEffect } from 'react';
 import styles from './Flashcard.module.css';
-import { FaQuestion, FaCommentDots, FaSave, FaCheck, FaTimes } from 'react-icons/fa';
 
-import { Button } from 'components';
+import {
+    Button,
+    CrossIcon,
+    CheckIcon,
+    SaveIcon,
+    AnswerIcon,
+    QuestionIcon,
+} from 'components';
 
-export function CardContent({ question, addToUserBase, known, unknown}) {
-    const [isKnownAdded, setIsKnownAdded] = useState('iconButton');
-    const [isUnknownAdded, setIsUnknownAdded] = useState('iconButton');
+export function CardContent({ question, addToUserBase, known, unknown }) {
+    const [isKnownAdded, setIsKnownAdded] = useState('#416071');
+    const [isUnknownAdded, setIsUnknownAdded] = useState('#416071');
 
     useEffect(() => {
         isClicked(known, setIsKnownAdded);
         isClicked(unknown, setIsUnknownAdded);
     });
 
     const isClicked = (base, setClass) => {
-        if(base.includes(question.id)){
-            setClass('iconButton iconButtonActive')
-        }else{
-            setClass('iconButton')
+        if (base.includes(question.id)) {
+            setClass('#9e005d');
```

Comment:
same here, setClass tells there should be a className

Reply:
I had the same thought when I was looking at this, you should rename these functions and constants.